ui/admin/app/components/worker-diagram/dual-filter/index.hbs

```diff
@@ -0,0 +1,67 @@
+<div data-test-dual-filters>
```

Comment:
If the only reason this exists is to provide a data attribute, I think we should find another way.  Otherwise, we're implying that all components will always need wrappers.

Reply:
Removed.